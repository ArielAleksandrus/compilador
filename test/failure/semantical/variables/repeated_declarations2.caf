programa{
	se(int a = 3 <= int a = 4) entao
		novalinha;
}