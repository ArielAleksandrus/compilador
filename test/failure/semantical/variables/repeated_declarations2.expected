ERROR => Line 2, col 22: Redeclaration of variable. 'a' previously found in line 2, col 9
